src/itLessConfig.js

```diff
@@ -0,0 +1,4 @@
+export const isEphem = insights.chrome.getEnvironment() === 'ephem';
```

Comment:
This is a bit different from our `chrome.isFedramp` helper as we will use this to lookup the correct invite API base URL (from a static file in each env).